src/main/java/com/peerrequest/app/api/DirectRequestsController.java

```diff
@@ -210,6 +265,21 @@ public DirectRequest.Dto patchDirectRequest(@RequestBody final DirectRequest.Dto
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "state must be set");
         }
 
+        if (updater.state().get() == DirectRequest.RequestState.PENDING) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "state can not be set to pending");
+        }
+
+        if (updater.state().get() == DirectRequest.RequestState.ACCEPTED) {
+
+            Long entryId = this.directRequestProcessService.get(request.get().getDirectRequestProcessId()).get()
+                    .getEntryId();
+
+            Review.Dto review = new Review.Dto(Optional.empty(), Optional.of(reviewerId), Optional.of(entryId),
+                    Optional.empty(), Review.ConfidenceLevel.LOW, null, null, null, null, null, null, null);
+
```

Comment:
if case for Declined is missing. Notification must be send to researcher. Add a \\todo: here